portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -31,6 +31,19 @@
   private static final String ENTITY_PROPERTY_KEY_6 = "address";
   private static final String ENTITY_PROPERTY_KEY_7 = "description";
 
+  class Result {
+    private StoreStatus status;
```

Comment:
Why do you have two enums, can you have a single Enum which the value contains the success and all the error cases.